File: shopica-admin/src/app/modules/inventory/models/purchase-order.ts
import { OrderStatus } from "@app/modules/order/model/order";
import { PurchaseOrderDetail } from "./purchase-order-detail";

export interface PurchaseOrder {
    id?: number;
    totalPrice: number;
    orderDate: Date;
    status: OrderStatus;
    purchaseOrderDetails: PurchaseOrderDetail[];
}

File: shopica-services/src/Services/Inventory/Inventory.API/Models/PurchaseOrder.cs
using Inventory.API.Models.Enums;

namespace Inventory.API.Models
{
    public class PurchaseOrder : BaseEntity
    {
        public int Id { get; set; }
        public decimal TotalPrice { get; set; }
        public DateTime OrderDate { get; set; }
        public OrderStatus Status { get; set; }
        public IEnumerable<PurchaseOrderDetail> PurchaseOrderDetails { get; set; }
    }
}
